en/docs/tutorials/connect-your-existing-ballerina-project-to-choreo.md

```diff
@@ -41,6 +41,33 @@ Read and write access to code, pull requests, and repository hooks
 10. Review and click **Merge pull request**, and then click **Confirm Merge**. 
    When you merge the pull request, it adds the necessary metadata files to connect your Ballerina project repository to Choreo so that you can proceed to create the component.
 
+11. To configure the pre-commit hook that Choreo uses to extract configurables defined in the Ballerina code, follow the steps given below:
+
+    !!! info
+        When you develop a Choreo component via the Web Editor, Choreo configures this pre-commit hook automatically. However, when you develop a component locally, you need to configure this pre-commit hook manually.
+
+     1. Carry out a GitHub pull locally to update your clone with the latest changes.
+
+     2. In your terminal, navigate to the root directory of your GitHub repository and issue the following commands:
```

Comment:
Before going to issue these commands, user should check whether the pre-commit hook file is exists in the give path. otherwise this is not working.

Reply:
Added a note for this.